src/utils/protocol.py

```
@@ -166,46 +183,107 @@ def metals_treatment_block(toxins_data: list[dict]):
+                {
+                    "display_name": metal.lower(),
+                    "freq": toxin_content.get("frequency"),
+                    "time": toxin_content.get("time"),
+                }
+            )
 
-        }
-    ]
-    """
+        else:
+            for k, v in toxins_metadata.items():
```

Ruff (0.14.14)

[warning] Loop control variable `k` not used within loop body

Rename unused `k` to `_k`

(B007)